packages/application-config/src/load-deprecated-config.ts

```diff
@@ -0,0 +1,70 @@
+import type { CspDirective } from './schema';
+import type { ApplicationConfig } from './types';
+
+import fs from 'fs';
+import substituteEnvVariablePlaceholders from './substitute-env-variable-placeholders';
+import { parseJsonFile } from './utils';
+
+export type DeprecatedOptions = {
+  envPath: string;
+  headersPath: string;
+  cspPath?: string;
+};
+
+// List the required fields of `env.json`
```

Comment:
`env.json` should be created by the users by themselves and no longer part of the repo. Right?

Reply:
I'm not 100% sure what you mean here but in general we want to start using the new config file (as we do now). I also need to update the documentation and write a release note (separate PR).
People can still use the `env.json` file, as long as it's supported. For example, in the next major version v17 we can drop support for that.

Does this answer your question?

Reply:
yes. That answers the question